buildkitd/buildkitd.go

```diff
@@ -423,12 +435,16 @@ func MaybePull(ctx context.Context, image string) error {
 	}
 	args = append(args, image)
 	cmd = exec.CommandContext(ctx, "docker", args...)
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
+	console.
+		WithPrefix("buildkitd-pull").
```

Comment:
Nit: the message also says it is pulling. We don't do separate prefixes for when other images are being pulled or pushed, why would we here?

Reply:
It's mainly because of the `...Done` message. There's one for pulling and one for starting.